lib/fog.rb

```diff
@@ -0,0 +1,3 @@
+module Fog
```

Comment:
WTF?

Reply:
I have the following error when uploading to s3 "uninitialized constant CarrierWave::Storage::Fog" without this module. That trick resolve problem.